libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using prompts = Microsoft.Bot.Builder.Prompts;
+using static Microsoft.Bot.Builder.Prompts.PromptValidatorEx;
+using Microsoft.Bot.Schema;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder.Prompts;
+
+namespace Microsoft.Bot.Builder.Dialogs
+{
+    public class OAuthPromptSettingsWithTimeout : prompts.OAuthPromptSettings
+    {
+        public int? Timeout { get; set; }
+    }
+
+    public class OAuthPromptState : PromptOptions
+    {
+        public OAuthPromptState(PromptOptions defaultPromptOptions) : base()
+        {
+            if (defaultPromptOptions != null)
```

Comment:
If I pass in null, what happens? 

I assume all the default prompt bits are set - but from this code that's not obvious.

Reply:
add an explanation to the doc comment

---
In reply to: [184817459](https://github.com/Microsoft/botbuilder-dotnet/pull/520#discussion_r184817459) [](ancestors = 184817459)